accounts/templates/accounts/onboarding_base.html

```diff
@@ -0,0 +1,65 @@
+{% load static %} {% load static tailwind_tags %}
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  {% csrf_token %}
+  <script src="https://kit.fontawesome.com/c60455fa26.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="{% static 'landing_page/css/base.css' %}" />
+  <link rel="stylesheet" href="{% static 'accounts/css/onboarding_base.css' %}" />
+  <link rel="stylesheet" href="{% static 'accounts/css/modal.css' %}" />
+  <title>ReeldIn</title>
```

Comment:
am now realizing we have just 'ReeldIn' as the title for every single page - we should change that.
here, just make this like 'ReeldIn: User Onboarding' or something